stats.go

```diff
@@ -382,7 +382,7 @@ func (h *histogram) snapshotValues() map[float64]int64 {
 
 	vals := make(map[float64]int64, len(h.buckets))
 	for i := range h.buckets {
-		vals[h.buckets[i].valueUpperBound] = h.buckets[i].samples.value()
+		vals[h.buckets[i].valueUpperBound] = h.buckets[i].samples.snapshot()
```

Comment:
This maps to https://sourcegraph.uberinternal.com/github.com/uber-go/tally/-/blob/stats.go#L102 since `h.buckets` here are `[]histogramBucket` and `histogramBucket.samples` are `*counter`. This looks correct vs. prior use of `value()`(https://sourcegraph.uberinternal.com/github.com/uber-go/tally/-/blob/stats.go#L73).

Reply:
yep, that's right. nice catch, thanks for fixing!

also: please use sourcegraph.com URLs for OSS projects.

Reply:
> also: please use sourcegraph.com URLs for OSS projects.

Derp. Corrected.